### ISSUE TITLE
TypeError: Cannot read property 'listen' of undefined new Layout .../node_modules/ra-ui-materialui/esm/layout/Layout.js:111

### ISSUE THREAD

**Issue by VovaGD** [marmelab/react-admin#4288]
May be somebody can help me? Very frustrating especially on the very beginning of Tutorial.
I'm getting following error just on the beginning of https://marmelab.com/react-admin/Tutorial.html
On step when I've just added line "< Resource name="users" list={ ListGuesser } / >"

TypeError: Cannot read property 'listen' of undefined
new Layout
d:/js/jwhlp1/node_modules/ra-ui-materialui/esm/layout/Layout.js:111
 108 |  *
 109 |  * @see https://stackoverflow.com/questions/48121750/browser-navigation-broken-by-use-of-react-error-boundaries
 110 |  */
 111 | props.history.listen(function () {
 112 |     if (_this.state.hasError) {
 113 |         _this.setState({ hasError: false });
 114 |     }

**Comment by djhi** [comment]
Thanks for reporting this. If you are able to illustrate the bug or feature request with an example, please provide a sample application via one of the following means:

* CodeSandbox (https://codesandbox.io/s/github/marmelab/react-admin/tree/master/examples/simple)
* A sample application via GitHub

**Comment by VovaGD** [comment]
Thank you for your attention! 
I've tried reproduce in CodeSandbox, but it works fine there.
So, then I've added repo https://github.com/VovaGD/react-admin-tmp/blob/master/src/  (check README.md, I tried explain my steps that I performed on 2 different computers, - probably main reason is in that steps, since it works fine in CodeSandbox. By the way I've tried couple months before as far as I remember same steps and it worked fine.
